e2e/tests/childSolicitors_test.js

```diff
@@ -92,9 +128,27 @@ Scenario('HMCTS assign a different solicitor for some of the children', async ({
   caseViewPage.selectTab(caseViewPage.tabs.casePeople);
 
   for (const [index, child] of children.entries()) {
-    const solicitor = index === childWithDifferentRegisteredSolicitorIdx ? alternativeSolicitor : (index === childWithUnregisteredSolicitorIdx ? unregisteredSolicitor : mainSolicitor);
+    const solicitor = index === childWithDifferentRegisteredSolicitorIdx ? solicitor2 : (index === childWithUnregisteredSolicitorIdx ? unregisteredSolicitor : solicitor1);
     assertChild(I, index + 1, child.value, solicitor);
   }
+
+  caseViewPage.selectTab(caseViewPage.tabs.changeOfRepresentatives);
+  assertChangeOfRepresentative(I, 1, 'FPL', `${children[childWithDifferentRegisteredSolicitorIdx].value.party.firstName} ${children[childWithDifferentRegisteredSolicitorIdx].value.party.lastName}`, 'HMCTS', {removedUser: solicitor1.details, addedUser: solicitor2.details });
+  assertChangeOfRepresentative(I, 2, 'FPL', `${children[childWithUnregisteredSolicitorIdx].value.party.firstName} ${children[childWithUnregisteredSolicitorIdx].value.party.lastName}`, 'HMCTS', {removedUser: solicitor1.details, addedUser: unregisteredSolicitor.details });
```

Comment:
`caseViewPage.selectTab(...)` doesn't include a wait after the click action, so it'd be more robust to wait for the tab to load before asserting the content, either here or within `assertChangeOfRepresentative(...)`.
Worst case, there's very little time penalty to doing the check when the content's already loaded